ci/docker/data6.build.wanda.yaml

```diff
@@ -0,0 +1,15 @@
+name: "data6build"
+froms: ["cr.ray.io/rayproject/oss-ci-base_ml"]
+dockerfile: ci/docker/data.build.Dockerfile
+srcs:
+  - ci/env/install-dependencies.sh
+  - python/requirements.txt
+  - python/requirements_compiled.txt
+  - python/requirements/test-requirements.txt
+  - python/requirements/ml/dl-cpu-requirements.txt
+  - python/requirements/ml/data-requirements.txt
+  - python/requirements/ml/data-test-requirements.txt
+build_args:
+  - ARROW_VERSION=6.*
```

Comment:
do these really need to be built 3 times?

where is `ARROW_VERSION` being used?

Reply:
It's just a version of one single package https://github.com/ray-project/ray/blob/master/ci/env/install-dependencies.sh#L408; but yeah that simple thing cause 3 builds, unless the version can be switched at runtime

These 3 builds have a big shared layer but due to wanda caching I don't bother restructure this too much

Let me know if you have some other brilliant ideas!

Reply:
ok, let's use this for now, and maybe iterate later.

interesting that we just keep expanding our wanda bases..

Reply:
I gonna blame the caching feature ;))